src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/MediationCSVService.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.civil.service.citizenui;
+
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.InputStreamSource;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.sendgrid.EmailAttachment;
+import uk.gov.hmcts.reform.civil.sendgrid.EmailData;
+
+import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MediationCSVService {
+
+    private final String SITE_ID = "5";
+    private final String CASE_TYPE = "1";
+    private final String CHECK_LIST = "4";
+    private final String PARTY_STATUS = "5";
+    private final String toEmail = "smallclaimsmediation@justice.gov.uk";
+    private final String subject = "OCMC Mediation Data";
+    private final String filename = "ocmc_mediation_data.csv";
+
+    public Optional<EmailData> prepareEmail(CaseData data) {
+        String content = generateAttachmentContent(data);
```

Comment:
this is csv service. perhaps it should not prepare emails instead all it should do is generate the attachment. i.e. csv file

Reply:
so its method that we are going to use will be generateCSVFile (and would do what you generateAttachmentContent is doing)